### DISCUSSION TITLE
Webshare proxies throwing Auth method unacceptable `[255]` error

### DISCUSSION THREAD

**Discussion by kishoreKlkr** [reacherhq/check-if-email-exists#1461] [Q&A]
I deployed the docker to AWS ec2 machine, where the validation works fine, when I run via proxy IP, I get this error
"smtp":{"error":{"type":"SocksError","message":"Auth method unacceptable `[255]`."}}

please suggest the actions I can take.
I am using webshare IPs which appears to be SOCKS5 supporting. But its not working

**Comment by zieddhf** [comment]
I'm having same issue here, @kishoreKlkr did you find a solution?

**Comment by email-verification-free-code** [comment]
> I deployed the docker to AWS ec2 machine, where the validation works fine, when I run via proxy IP, I get this error "smtp":{"error":{"type":"SocksError","message":"Auth method unacceptable `[255]`."}}
> 
> please suggest the actions I can take. I am using webshare IPs which appears to be SOCKS5 supporting. But its not working

You need a specialist proxy service for this like https://www.proxy4smtp.com/
